emails/tests/utils_tests.py

```diff
@@ -199,11 +223,17 @@ def test_simple_general_tracker_replaced_with_relay_content(self):
             '<a href="https://open.tracker.com/foo/bar.html">A link</a>\n'
             + '<img src="https://open.tracker.com/foo/bar.jpg">An image</img>'
         )
-        changed_content, tracker_details = remove_trackers(content)
+        from_address = "spammer@email.com"
+        datetime_now = "1682472064"
+        changed_content, tracker_details = remove_trackers(
+            content, from_address, datetime_now
+        )
```

Comment:
nit: move these up to the class level, so that the code becomes:

```suggestion
        changed_content, tracker_details = remove_trackers(
            content, self.from_address, self.datetime_now
        )
```

Making the same change to other tests will remove some redundant setup